plugins/external/skeletons/python/plugin-with-feedback.py

```diff
@@ -6,6 +6,7 @@
 'omprog' action like the following:
     action(type="omprog"
         binary="/usr/bin/myplugin.py"
+        output="/var/log/myplugin.log"
```

Comment:
This must not be written to /var/log, as CI may not have write access. It should start with the value from env-var ```$RSYSLOG_DYNNAME``` - this is part to support parallel make check.

Reply:
Note that this is not test code, it is just an example comment in the example Python skeleton (explaining how omprog would be typically configured).